frontend-react/package.json

```
@@ -95,51 +95,51 @@
 		"@dnd-kit/core": "^6.3.1",
 		"@dnd-kit/utilities": "^3.2.2",
 		"@popperjs/core": "^2.11.8",
-		"bootstrap": "^5.3.3",
-		"bootstrap-icons": "^1.11.3",
+		"bootstrap": "^5.3.8",
+		"bootstrap-icons": "^1.13.1",
 		"clsx": "^2.1.1",
-		"nats.ws": "^1.29.2",
-		"react-bootstrap": "^2.10.6",
-		"react-dom": "^19.0.0",
-		"react-error-boundary": "^4.1.2",
-		"react-router-dom": "^7.0.2",
-		"zod": "^3.23.8"
+		"nats.ws": "^1.30.3",
+		"react-bootstrap": "^2.10.10",
+		"react-dom": "^19.2.0",
+		"react-error-boundary": "^6.0.0",
+		"react-router-dom": "^7.9.4",
+		"zod": "^4.1.12"
 	},
 	"peerDependencies": {
 		"react": "19"
 	},
 	"devDependencies": {
-		"@cucumber/cucumber": "^11.1.0",
-		"@eslint/js": "^9.16.0",
-		"@types/node": "^20.17.9",
-		"@types/react": "^19.0.1",
-		"@types/react-dom": "^19.0.1",
-		"@typescript-eslint/eslint-plugin": "^8.17.0",
-		"@typescript-eslint/parser": "^8.17.0",
-		"@vitejs/plugin-react-swc": "^3.7.2",
-		"@vitest/ui": "^2.1.8",
-		"eslint": "^9.16.0",
-		"eslint-config-prettier": "^9.1.0",
-		"eslint-import-resolver-typescript": "^3.7.0",
+		"@cucumber/cucumber": "^12.2.0",
+		"@eslint/js": "^9.38.0",
+		"@types/node": "^24.9.1",
+		"@types/react": "^19.2.2",
+		"@types/react-dom": "^19.2.2",
+		"@typescript-eslint/eslint-plugin": "^8.46.2",
+		"@typescript-eslint/parser": "^8.46.2",
+		"@vitejs/plugin-react-swc": "^4.2.0",
+		"@vitest/ui": "^4.0.3",
+		"eslint": "^9.38.0",
+		"eslint-config-prettier": "^10.1.8",
+		"eslint-import-resolver-typescript": "^4.4.4",
 		"eslint-plugin-jsx-a11y": "^6.10.2",
-		"eslint-plugin-prettier": "^5.2.1",
-		"eslint-plugin-react": "^7.37.2",
-		"eslint-plugin-react-hooks": "^5.1.0-rc.1",
+		"eslint-plugin-prettier": "^5.5.4",
+		"eslint-plugin-react": "^7.37.5",
+		"eslint-plugin-react-hooks": "^7.0.1",
 		"eslint-plugin-tsdoc": "^0.4.0",
-		"globals": "^15.13.0",
-		"happy-dom": "^15.11.7",
-		"prettier": "3.4.2",
-		"react": "^19.0.0",
-		"sass": "^1.82.0",
-		"typedoc": "^0.27.3",
-		"typedoc-plugin-markdown": "^4.3.1",
-		"typedoc-plugin-mdn-links": "^4.0.3",
-		"typedoc-plugin-zod": "^1.3.0",
-		"typescript": "^5.7.2",
-		"typescript-eslint": "^8.17.0",
-		"vite": "^6.0.3",
-		"vite-plugin-banner": "^0.8.0",
-		"vitest": "^2.1.8"
+		"globals": "^16.4.0",
+		"happy-dom": "^20.0.8",
+		"prettier": "^3.6.2",
```

Can we pin down prettier again?

```suggestion
		"prettier": "3.6.2",
```

Explanation from prettier docs:

> Install an exact version of Prettier locally in your project. This makes sure that everyone in the project gets the exact same version of Prettier. Even a patch release of Prettier can result in slightly different formatting, so you wouldn’t want different team members using different versions and formatting each other’s changes back and forth.

https://prettier.io/docs/install#summary